### DISCUSSION TITLE
Maximizing space with dock

### DISCUSSION THREAD

**Discussion by gallarotti** [rxhanson/Rectangle#947] [Ideas]
With multiple monitors on OS X the dock can be moved from screen to screen just by moving the mouse below the lower edge of a screen. Then the dock appears on that screen and stays there. Rectangle honors the size of the dock when maximizing only on the screen that has the dock when the window is maximized. Would it be possible to add an option to always leave space for the dock at the bottom even if the dock is not currently on that screen?

**Comment by rxhanson** [comment]
Is this what you are looking for?

https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#setting-gaps-at-the-screen-edges

**Reply by gallarotti** [reply]
Brilliant, thank you so much! I wasn't aware of these settings - maybe they should be advertised more within the application itself?
